readme.md

```diff
@@ -45,7 +60,89 @@ The codebase also has a number of reference examples.  These are all marked with
 # TODO: delete me; this is just a reference example
 ```
 
-# sample-django-app TODO
+
+# Feature Descriptions
+
+
+## Django-React (django_react)
+
+
+### Files
+
+The following files and folders are only needed if `django_react` is a desired feature in your app and can be safely
+deleted from projects which do not leverage the feature.
+
+- `nwb.config.js`
+- `package.json`
+- `webpack-stats.json`
+- `config/webpack_loader.py`
+- `src/`
+
+### Additional Setup
+
+
+When using this feature, make sure to install the Node.js requirements using the manager of your choice 
+(either `npm install` or `yarn install` will work) before proceeding with development.
+
+
+### Special Consideration for Running
+
+
+For development on localhost when using Django-React, you should run the following command in a separate terminal to
+your standard `runserver` command.
+    
+- `nwb serve --no-vendor`
+
+If you have configured everything correctly, you should see each command complete and notify you
+that the project is ready to be viewed.
+
+- If you include `nwb` as a dependency, you can use the locally-installed `nwb` by running `node_modules/.bin/nwb server --no-vendor` instead of relying on a globally installed `nwb`.
+
+
+### Adding a new React component
+
+
+In this paradigm, React components are compiled and injected into the standard Django template. This means we can take 
+advantage of the built-in templating functionality of Django and, with a bit of elbow grease, use the power of React to
+make those templates responsive.
+
+`django-react-loader` uses the same basic pattern for any component:
+
+1. First, ensure that the library is loaded in your template: `{% load django_react_components %}`
+2. Next, ensure that you have rendered the React runtime bundle: `{% render_bundle 'runtime' %}`
+   - Note that you only have to do this once per page where React components will be used. 
+3. Finally, load your React component on the page. `{% react_component 'Component' %}`
+    - You can add any number of props as named keywords, e.g. `{% react_component 'Home' id='home' prop1=value_from_context %}`
+    - You can also choose to pass props as an object instead of individual kwargs, e.g. `{% react_component 'Hello' id='hello' props=sample_props %}`.
+
+### Preparing for deployment
+
+
+The preferred option for deployment is to add the below compilation step to the deployment configuration rather than 
+building it locally. However, if you wish to build the app locally:
+
+- run `nwb build --no-vendor`. This will generate or replace a `webpack_bundles` folder in your `/static` folder populated with the compiled React components.
```

Comment:
Maybe add something like "This then allows `collectstatic` to collect these static assets and make them available via the usual static assets pipeline set up in the deploy configuration."